cli/compose/schema/data/config_schema_v3.5.json

```diff
@@ -395,7 +395,24 @@
       "type": "object",
       "properties": {
         "cpus": {"type": "string"},
-        "memory": {"type": "string"}
+        "memory": {"type": "string"},
+        "genericResources": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "discreteResourceSpec": {
```

Comment:
This should use underscores instead of CamelCase to match all the other fields: `generic_resources`, `discrete_resource_spec`.